src/vic3_world/pacts/pacts_importer.h

```diff
@@ -0,0 +1,20 @@
+#ifndef SRC_VIC3WORLD_PACTS_PACTSIMPORTER_H
+#define SRC_VIC3WORLD_PACTS_PACTSIMPORTER_H
+
+
+
+#include <istream>
+
+#include "src/vic3_world/pacts/pact.h"
+
+
+namespace vic3
+{
+
+std::map<int, Pact> ImportPacts(std::istream& input_stream);
```

Comment:
Add tests for this. Have one for the happy path and another verifying the broken pact code path.